posthog/hogql/database.py

```diff
@@ -278,6 +278,35 @@ def hogql_table(self):
         return "session_recording_events"
 
 
+class CohortPeople(Table):
```

Comment:
Tangential to this PR but I wonder what stops these definitions and the actual table definitions from diverging other than people being careful